R/sce_to_anndata.R

```diff
@@ -5,6 +5,9 @@
 #' @param anndata_file Path to output AnnData file. Must be an `.h5` or `.hdf5`
 #'
 #' @return original SingleCellExperiment object used as input (invisibly)
+#' **Note that any columns present in the `rowData` of an SCE object that contains
+# duplicated information, e.g. duplicate gene identifiers, are converted to
+# categorical data by the `anndata` package.
```

Comment:
Roxygen ignores comment lines without a single quote, so this change will require a rebuild to update the documentation.
```suggestion
#' duplicated information, e.g. duplicate gene identifiers, are converted to
#' categorical data by the `anndata` package.
```

Reply:
I was really hoping this was a transient failure. Fingers crossed it works this time. Thanks for taking a look!